sky/adaptors/common.py

```diff
@@ -60,6 +60,10 @@ def wrapper(*args, **kwargs):
                 m.load_module()
             return func(*args, **kwargs)
 
+        # Preserve lru_cache methods
```

Comment:
Instead of pumping the function, should we just reverse the order of `load_lazy_modules` and `lru_cache` functions in the adaptors?